Packs/ApiModules/Scripts/MLApiModule/TestData/input_json_file_test

```diff
@@ -0,0 +1,2 @@
+[{"body": "This is a sample of the body", "subject": "This is a sample of the subject"},
+{"body2": "This is a sample of the body2", "subject2": "This is a sample of the subject2", "subject": "This is a sample of the subject"}]
```

Comment:
maybe add also a CSV - we use it also in the train from file. Also include utf8 characters in the CSV